server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -665,9 +665,32 @@ export class TenantManager {
 	 * Retrieves all the raw database tenant documents
 	 */
 	private async getAllTenantDocuments(includeDisabledTenant = false): Promise<ITenantDocument[]> {
-		const allFound = await this.runWithDatabaseRequestCounter(async () =>
```

Comment:
What is this `runWithDatabaseRequestCounter` do? Should we wrap it with the new `getTenantDocumentsByBatch` when make query?